utils/deprecate_models.py

```diff
@@ -0,0 +1,472 @@
+
+                else_block = [
+                    [sub_block] if not isinstance(sub_block, list) else sub_block for sub_block in else_block
+                ]
+
+                # Sort the sub-blocks in the else block
+                else_block = [sorted(sub_block) for sub_block in else_block]
+
+                # Flatten the lists so they're all lines
+                else_block = [line for sub_block in else_block for line in sub_block]
+
+                # Add the else block to the file lines and reset it
+                new_init_file_lines.extend(else_block)
+                else_block = []
+                in_else_block = in_base_imports = False
+
+                # If we were in a maybe block, we now know it wasn't part of the else block
+                maybe_else_block.append(line)
+                new_init_file_lines.extend(maybe_else_block)
+                maybe_else_block = []
+
+            elif line.endswith(("[", "(")) and not indented_block:
+                # We're at the start of an indented block
+                indented_block.append(line)
+                open_indent_level = indent
+
+            elif indented_block:
+                if indent == open_indent_level and (
+                    line.strip().endswith(("]", ")")) or line.strip().startswith(("],", "),"))
+                ):
+                    # We're at the end of the indented block. Add the block to the sub-block and reset it
+                    indented_block.append(line)
+                    sub_block.append("\n".join(indented_block))
+                    indented_block = []
+                    open_indent_level = -1
+                else:
+                    # We're still in the indented block
+                    indented_block.append(line)
+
+            # We have a comment line - create a new sub-block so lines above the comment are grouped together when sorting
+            elif line.strip().startswith("#"):
+                if sub_block:
+                    else_block.append(sub_block)
+                    sub_block = []
+
+                # When adding else blocks to the file lines, we sort each sub-block, but not between sub-blocks.
+                # Comment lines are added directly to make sure they stay between the sub-blocks they're associated with
+                # structure: [sub_block1, comment1, sub_block2, comment2, ...]
+                else_block.append(line)
+
+            else:
+                sub_block.append(line)
+
+    # Add the last sub-block if it exists
+    if else_block:
+        # Sort the sub-blocks in the else block
+        else_block = [
+            [sorted(sub_block) if isinstance(sub_block, list) else sub_block for sub_block in sub_blocks]
+            for sub_blocks in else_block
+        ]
+        # Flatten the lists so they're all lines
+        else_block = [line for sub_block in else_block for line in sub_block]
+        new_init_file_lines.extend(else_block)
+
+    return "\n".join(new_init_file_lines)
```

Comment:
🤦‍♀️  I didn't know this existed - it would have saved me a lot of strife! Thanks for highlighting - and definitely in the camp of killing your darlings. I've removed this logic and just used the existing functionality